nodejs/week3/Mealsharing/src/backend/api/meals.js

```diff
@@ -0,0 +1,202 @@
+const knex = require("../database");
+const express = require("express");
+const router = express.Router();
+//get method
+router.get("/meals", async (req, res) => {
```

Comment:
You only need to register one request handler for the `/meals` path and it should be able to handle the diverse query params: `maxPrice`, `limit`, etc. You can remove this one and keep the one below.